ibm_catalog.json

```diff
@@ -285,9 +304,91 @@
 							"default_value": "__NULL__",
 							"description": "The CRN of an existing KMS instance to use in this solution. If not set, a new KP instance is provisioned.",
 							"required": false
+						},
+						{
+							"key": "existing_en_instance_crn",
+							"type": "string",
+							"default_value": "__NULL__",
+							"description": "The CRN of an existing event notification instance to use in this solution. If not set, a new event notification instance is provisioned.",
+							"required": false
+						},
+						{
+							"key": "elasticsearch_plan",
+							"type": "string",
+							"default_value": "enterprise",
+							"description": "The pricing plan for the Databases for Elasticsearch instance.",
+							"required": false,
+							"options": [
+								{
+									"displayname": "Enterprise",
+									"value": "enterprise"
+								},
+								{
+									"displayname": "Platinum",
+									"value": "platinum"
+								}
+							]
+						},
+						{
+							"key": "existing_discovery_instance",
+							"type": "string",
+							"default_value": "__NULL__",
+							"description": "The CRN of an existing WatsonX SaaS discovery instance to use in this solution. If not set, a new discovery instance is provisioned depending on which plan is selected.",
+							"required": false
+						},
+						{
+							"key": "existing_assistant_instance",
+							"type": "string",
+							"default_value": "__NULL__",
+							"description": "The CRN of an existing WatsonX SaaS assistant instance to use in this solution. If not set, a new assistant instance is provisioned depending on which plan is selected.",
+							"required": false
+						},
+						{
+							"key": "existing_governance_instance",
+							"type": "string",
+							"default_value": "__NULL__",
+							"description": "The CRN of an existing WatsonX SaaS governance instance to use in this solution. If not set, a new governance instance is provisioned depending on which plan is selected.",
+							"required": false
+						},
+						{
+							"key": "existing_studio_instance",
+							"type": "string",
+							"default_value": "__NULL__",
+							"description": "The CRN of an existing WatsonX SaaS studio instance to use in this solution. If not set, a new studio instance is provisioned depending on which plan is selected.",
+							"required": false
+						},
+						{
+							"key": "existing_machine_learning_instance",
+							"type": "string",
+							"default_value": "__NULL__",
+							"description": "The CRN of an existing WatsonX SaaS machine learning instance to use in this solution. If not set, a new machine learning instance is provisioned depending on which plan is selected.",
+							"required": false
+						},
+						{
+							"key": "existing_db_instance_crn",
```

Comment:
suggest to change db to elastic_search here to be very explicit. Also be consistent with output name which is elasticsearch_crn

Reply:
we will address these changes in dev repo (to not increase merge conflicts) with new PR. Issue created https://github.ibm.com/GoldenEye/issues/issues/10462